addons/packages/cartographer/0.2.2/README.md

```diff
@@ -0,0 +1,120 @@
+# Package for Cartographer
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Overview](#overview)
+- [Pre-requisites](#pre-requisites)
+- [Installation](#installation)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+[carvel]-based Packaging of [Cartographer].
+
+## Pre-requisites
+
+- [kapp-controller]
+- [Tanzu CLI]
+- [cert-manager]
```

Comment:
> _It seems cartographer has some dependencies on other packages being available in the cluster (I could spot at least cert-manager) It would be ideal to have this pre-requisite documented_

Reply:
similarly to `harbor`, added a section to the installation where `tanzu package install` for cert-manager is performed before `cartographer`